study.md

```diff
@@ -44,21 +44,23 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+Developers define 'views' as individual HTML fragments that perform certain functions.  Users might see them as separate pages, like an order entry screen or a product search page.  Multiple views can exist on one HTML page.  The 'view-state' refers to the view(s) that are currently showing up on the UI for the user.  Routers are mechanism that changes views, thus allowing users to change views while not leaving the page
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+Some of the advantages of an SPA over a traditional web application are
+more fluid functionality, getting ride of many page reloads, and transferring data rather than pages or markup.  With AJAX and JSON,
+actions that would normally require multiple different html pages and many different page reloads and changes could all be completed on one single markup page.
 ```
 
 ## List some technologies used to make SPA's
 
 What are some of the tools used to create single page applications?
 
 ```md
-<!-- your answer here -->
+AJAX and JSON are the primary tools behind SPAs.  AJAX sends requests to get data in JSON format from the server, essentially treating the server like an API.  This is the mechanism that allows us to change views on the same page.  Other JavaScript frameworks like angular are commonly used in SPAs as well.
```

Comment:
Yes front end frameworks play a large role in SPAs because they allow you to easily manage front end routing.